src/nunavut/cli/__init__.py

```diff
@@ -562,9 +562,7 @@ def main() -> int:
     #
     extra_includes = args.lookup_dir
 
-    # legacy variable. We'll support this for a time.
-    extra_includes_from_env = _extra_includes_from_env("UAVCAN_DSDL_INCLUDE_PATH")
-    extra_includes_from_env += _extra_includes_from_env("DSDL_INCLUDE_PATH")
+    extra_includes_from_env = _extra_includes_from_env("DSDL_INCLUDE_PATH")
```

Comment:
This might add unnecessary redundancy next to the `CYPHAL_PATH` environment variable that has become an almost de-facto standard already, as it is used in PyCyphal, Yakut, and Yukon and may be used by other implementations eventually. It contains the list of directories that contain DSDL root namespaces in them, separated with either `:` or `;`.

We may want to start using the same environment variable in Nunavut, such that the entries pulled through this variable are added after the entries given through the command line.

If we don't want to rely on the same env var in Nunavut, then at least `DSDL_INCLUDE_PATH` should be renamed to follow some consistent naming pattern, and the word "include" is better be removed since there is no concept of inclusion in DSDL.

Reply:
I'll just support them all.